src/ansys/dpf/post/simulation.py

```diff
@@ -0,0 +1,524 @@
+        >>> simulation = post.load_simulation(examples.static_rst)
+        >>> simulation.plot() # doctest: +NORMALIZE_WHITESPACE +ELLIPSIS
+        """
+        plt = DpfPlotter()
+        if mesh:
+            plt.add_mesh(self.mesh._meshed_region)
+        if constructed_geometries:
+            for geom in self.geometries:
+                getattr(plt, "add_" + str(type(geom).__name__).lower())(geom)
+        if loads:
+            pass
+        if boundary_conditions:
+            pass
+        plt.show_figure()
+
+    @property
+    def active_selection(self) -> Selection:
+        """Active selection used by default for result queries.
+
+        Returns a :object:`ansys.dpf.post.selection.Selection` object.
+
+        Examples
+        --------
+        >>> from ansys.dpf import post
+        >>> from ansys.dpf.post import examples
+        >>> simulation = post.load_simulation(examples.static_rst)
+        >>> selection = post.selection.Selection()
+        >>> simulation.activate_selection(selection=selection)
+        >>> print(simulation.active_selection) # doctest: +NORMALIZE_WHITESPACE +ELLIPSIS
+        <ansys.dpf.post.selection.Selection object at ...>
+        """
+        return self._active_selection
+
+    def activate_selection(self, selection: Selection):
+        """Sets a selection as active on the simulation.
+
+        Activating a given selection on a simulation means it is used
+        as a default selection/filter in further result queries.
+
+        Examples
+        --------
+        >>> from ansys.dpf import post
+        >>> from ansys.dpf.post import examples
+        >>> simulation = post.load_simulation(examples.static_rst)
+        >>> selection = post.selection.Selection()
+        >>> simulation.activate_selection(selection=selection)
+        >>> print(simulation.active_selection) # doctest: +NORMALIZE_WHITESPACE +ELLIPSIS
+        <ansys.dpf.post.selection.Selection object at ...>
+        """
+        self._active_selection = selection
+
+    def deactivate_selection(self):
+        """Deactivate the currently active selection.
+
+        Examples
+        --------
+        >>> from ansys.dpf import post
+        >>> from ansys.dpf.post import examples
+        >>> simulation = post.load_simulation(examples.static_rst)
+        >>> selection = post.selection.Selection()
+        >>> simulation.activate_selection(selection=selection)
+        >>> print(simulation.active_selection) # doctest: +NORMALIZE_WHITESPACE +ELLIPSIS
+        <ansys.dpf.post.selection.Selection object at ...>
+        >>> simulation.deactivate_selection()
+        >>> print(simulation.active_selection) # doctest: +NORMALIZE_WHITESPACE +ELLIPSIS
+        None
+        """
+        self._active_selection = None
+
+    @property
+    def _time_frequencies(self):
+        """Description of the temporal/frequency analysis of the model."""
+        return self._model.metadata.time_freq_support
```

Comment:
A private property? a mistake perhaps?

```suggestion
    @property
    def time_frequencies(self):
        """Description of the temporal/frequency analysis of the model."""
        return self._model.metadata.time_freq_support
```

Reply:
I think this was intentional, it is used in the public property "time_freq_support".